### ISSUE TITLE
Passing connect array fail

### ISSUE THREAD

**Issue by shannonsands** [algolia/react-nouislider#30]
I have a slider with two handles, and I need to have the connect option set to "true" for the lower handle, so it connects to the left edge. This works fine with a single handle and passing in "lower". I've tried commenting out the propType check and passing through values directly to nouislider, and it'll work for the single handle, but not for 2 or more. I just get "noUiSlider: 'connect' option doesn't match handle count." regardless of the number of values in the array. Any ideas?

**Comment by vvo** [comment]
@shannonsands Can you try to have this working with the original https://github.com/leongersen/noUiSlider project and then have the same thing with react-nouislider?

Once you have this working, compare the options you pass to the options we pass in our own code and see what happens. Thanks.

**Comment by shannonsands** [comment]
It works fine on noUISlider on it's own. For whatever reason the array was getting passed through with an extra element. I just modified noUiSlider itself to remove the array length check - it's unnecessary anyway IMO. So now it's all working fine.

I also noticed that they recently made an update that removed the 'lower' option along with a bunch of other changes: https://github.com/leongersen/noUiSlider#changelog. So your code probably wasn't the issue at all.
